hw/platform/snowy/snowy_rtc.c

```diff
@@ -260,7 +260,7 @@ void RTC_WKUP_IRQHandler(void)
     if(RTC_GetITStatus(RTC_IT_WUT) != RESET)
     {
         RTC_ClearITPendingBit(RTC_IT_WUT);
-        DRV_LOG("RTC", APP_LOG_LEVEL_DEBUG, "RTC WAKE IRQ");
+        printf("RTC WAKE IRQ");
```

Comment:
any reason why DRV_LOG doesn't work here?

Reply:
nope, thats an artifact of trying to fixup logging. Will revert